content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -385,6 +387,117 @@ psql file.sql
+
+   ```prisma file=schema.prisma
+   model User {
+     id    Int    @id @default(autoincrement())
+     email String @unique
+     name  String
+     posts Post[]
+   }
+   
+   model Post {
+     id        Int     @id @default(autoincrement())
+     title     String
+     content   String
+     published Boolean
+     user      User    @relation(fields: [userId], references: [id])
+     userId    Int
+   }
+   ```
+
+2. Execute `npx @snaplet/seed init` to introspect your database. This command generates the seed client and creates a `seed.ts` file.
+3. Use the generated `@snaplet/seed` client within the `seed.ts` file to compose your seeds.
+
+Here's how to create new users and posts by modifying the initially generated `seed.ts` file:
+
+```ts file=seed.ts highlight=14-20;normal
+/**
+ * ! Executing this script will delete all data in your database and seed it with 10 versions.
+ * ! Make sure to adjust the script to your needs.
+ * Use any TypeScript runner to run this script, for example: `npx tsx seed.ts`
+ * Learn more about the Seed Client by following our guide: https://docs.snaplet.dev/seed/getting-started
+ */
+import { createSeedClient } from '@snaplet/seed'
+
+const seed = await createSeedClient()
+
+// Truncate all tables in the database
+await seed.$resetDatabase()
+
+// Seed the database with 10 User
+await seed.User((x) =>
+  x(10, {
+    // Create 10 posts for each of those users
+    Post: (x) => x(10),
+  })
+)
+
+process.exit()
+```
+
+4. Add `tsx` (or any other typescript runners) as a development dependency:
+
+   ```
+   npm install -D tsx
+   ```
+
+5. Insert the `prisma.seed` field into your `package.json` file and configure commands to seed your database:
+
+   ```json file=package.json highlight=5;normal
+   {
+     "name": "my-project",
+     "version": "1.0.0",
+     "prisma": {
+       "seed": "npx tsx seed.ts"
+     },
+     "devDependencies": {
+       "@types/node": "^14.14.21",
+       "ts-node": "^9.1.1",
+       "typescript": "^4.1.3"
+     }
+   }
+   ```
+
+6. To seed the database, execute the following CLI command:
+
+   ```
+   npx prisma db seed
+   ```
+
+7. To ensure the client remains synchronized with your schema, add a `migrate` and `postmigrate` scripts to the package.json:
+
+   ```json file=package.json highlight=8-9;normal
+   {
+     "name": "my-project",
+     "version": "1.0.0",
+     "prisma": {
+       "seed": "npx tsx seed.ts"
+     },
+     "scripts": {
+      "migrate": "prisma migrate dev",
+      "postmigrate": "npx @snaplet/seed sync"
+    }
+     "devDependencies": {
+       "@types/node": "^14.14.21",
+       "ts-node": "^9.1.1",
+       "typescript": "^4.1.3"
+     }
+   }
+   ```
+
+   Now, instead of `npx prisma migrate dev` you can run `npm run migrate` which will also sync the seed
+   client after schema changes and keep both your database and seed client in sync.
```

Comment:
```suggestion
   Now, instead of `npx prisma migrate dev` you can run `npm run migrate` which will also sync the seed client after schema changes and keep both your database and seed client in sync.
```